.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug report (反馈 Bug)
+about: Create a bug report to help HugeGraph improve
+title: '[BUG] short description'
```

Comment:
improve "short description", if it to be filled, like "Type summary description here" is ok